src/components/ViewEditor.js

```diff
@@ -57,20 +78,41 @@ class ViewEditor extends React.Component {
 		) {
 			this.changeFile().then(() => console.log("Updated view file"));
 		}
+		if (prevProps.newUser !== this.props.newUser) {
+			this.setState({cardTourStart: this.props.newUser});
+		  }
 	};
 	
+	onStepsExit = () => {
+		this.setState(() => ({ cardTourStart: false}));
+	};
+
 	render = () => {
-		if (this.state.fileType === FILE_TYPE.CARD_VIEW) {
-			return <CardDeck data={this.state.data}/>;
-		} else if (this.state.fileType === FILE_TYPE.TEXT_VIEW) {
-			return <TextView data={this.state.data}/>;
-		}
-		return null;
+		return(
+			<div>
+				<Steps
+					enabled={this.state.cardTourStart && this.props.tagEditorOpened}
+					steps={this.state.steps}
+					initialStep={0}
+					onExit={this.onStepsExit}
+					options={{disableInteraction: true, exitOnOverlayClick: false, exitOnEsc: false}}
+				/>
+				{(this.state.fileType === FILE_TYPE.CARD_VIEW) ?
+					<CardDeck data={this.state.data}/> : 
+					(this.state.fileType === FILE_TYPE.TEXT_VIEW) ? 
```

Comment:
nested ternary operations aren't very readable - we should stick with the if/else block like before

Reply:
Conditional Rendering in react 😓. This is the only way unless we make a component

Reply:
Pitfalls of React render method, i cant use if else 😢